packages/terra-action-header/src/ActionHeader.jsx

```diff
@@ -47,7 +47,7 @@ const propTypes = {
    */
   onPrevious: PropTypes.func,
   /**
-   * Child element to be displayed on the right end of the header.
+   * Displays a single child element on the right end of the header.
```

Comment:
We should double check with @dkasper-was-taken to make sure this is what he had in mind. It's difficult to tell from the issue if he wanted the collapsable item view called out here specifically as the only element intended to be used.

Reply:
Yes, this should be clarified that the only intended component for that location is the terra-collapsible-menu-view, as currently consumers are trying to put other elements in their without realizing this.

Reply:
Ok @dkasper-was-taken, I will update it to just single terra-collapsible-menu-view. I assumed it could take any single terra component.